README.md

```diff
@@ -213,31 +212,31 @@ import Hello from '@/components/hello.vue'
 ```
 4. Add rules to your `tslint.json` and they will be applied to Vue files. For example, you could apply the Standard JS rules [tslint-config-standard](https://github.com/blakeembrey/tslint-config-standard) like this:  
 
-```json
```

Comment:
Isn't json  the correct choice here?

Reply:
I think it's better to keep it consistent and use `js` even here, to make code highlight identical in all code samples where JSON is involved.

Reply:
Where it is JSON rather than JS isn't highlighting as JSON a more helpful choice?  The distinction seems like it could be helpful.

Reply:
Sure it could. However, in this case the only distinction between `js` and `json` is the color of colons, so..

Can you change it to `json` or should I add another commit? 🙂

Reply:
65b2503 🙂